generator/fluentReportsGenerator.js

```diff
@@ -3403,11 +3457,11 @@ class frElement { // jshint ignore:line
             return;
         }
 
-        this._selected();
+        this._selected(e);
     }
 
     _blur(args) {
-        this._report.currentSelected = null;
+        this._report.currentSelected[0] = null;
```

Comment:
Can't this element be anyone of the elements in the currentSelect array -- so use splice to remove its actual index out of the array

Reply:
Oh, good catch. I was trying to find out where it was setting it to null, so I just added a bunch of redundancies to skip over the elem when they are null.